ImageFeed/ImagesList/ImagesListViewController.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  ViewController.swift
+//  ImageFeed
+//
+//  Created by Тимофей Спирин on 09.12.2024.
+//
+
+import UIKit
+
+class ImagesListViewController: UIViewController {
+    
+    @IBOutlet private var table: UITableView!
+    
+    private let photosName: [String] = Array (0..<20).map{ "\($0)" }
+    
+    private lazy var dateFormatter: DateFormatter = {
+        let formatter = DateFormatter()
+        formatter.dateStyle = .long
+        formatter.timeStyle = .none
+        return formatter
+    }()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        table.rowHeight = 200
+        table.contentInset = UIEdgeInsets(top: 12, left: 0, bottom: 12, right: 0)
+    }
+    //    MARK: - Class
+    
+}
+
+// MARK: -
```

Comment:
Добавление // MARK: - UITableViewDataSource поможет лучше структурировать код и упростит навигацию 

```suggestion
// MARK: -
```